hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/UtilitiesTestBase.java

```diff
@@ -155,6 +155,11 @@ public static void initTestServices(boolean needsHdfs, boolean needsHive, boolea
       zookeeperTestService = new ZookeeperTestService(hadoopConf);
       zookeeperTestService.start();
     }
+
+    jsc = UtilHelpers.buildSparkContext(UtilitiesTestBase.class.getName() + "-hoodie", "local[2]");
```

Comment:
should we make a constant for spark master local[4] for test cases?

Reply:
I think we should find a way to standardize on how to initialize spark for testing. There are a few patterns in the repo like bring up and teardown per test method vs per test class and which settings to add. Most are very similar so we can do that as a follow up.